README.md

```diff
@@ -8,142 +8,179 @@
 
 Documentation is available at http://docs.elemeno.io
 
-## API Overview
+## Usage
+
+```js
+var Elemeno = require('elemeno');
+
+var elemeno = new Elemeno('123e4567-e89b-12d3-a456-426655440000');
+
+elemeno.getCollectionItems('recipes', function(err, response) {
+	if (err) return console.log(err);
+
+	// Do something with the response
+	console.log(response);
+});
+```
+
+## Caching
+
```

Comment:
What are your thoughts on adding a note? Something like

**Note: The response from the Elemeno API is cached for 2 minutes, separately from this client library. Please take this into consideration**

I realize it is weird, but do you think it would be better than not mentioning that?